docs/core/tools/dotnet-package-search.md

```diff
@@ -0,0 +1,111 @@
+---
+title: dotnet package search command
+description: Searches a given source using the query string provided. If no sources are specified, all sources defined in the NuGet.Config file are used
+author: Nigusu-Allehu
+ms.date: 10/26/2023
+---
+# dotnet package search
+
+**This article applies to:** ✔️ .NET 8.0.2xx SDK and later versions
+
+## Name
+
+`dotnet package search` - Searches for a NuGet package.
+
+## Synopsis
+
+```dotnetcli
+dotnet package search [search terms] [options]
+    [--source <SOURCE>]
+    [--exact-match]
+    [--prerelease]
+    [--interactive]
+    [--take <TAKE>]
+    [--skip <SKIP>]
```

Comment:
See [dotnet tool search](https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-tool-search) for an example of how to format the Synopsis section. Also we generally try to put the options in alphabetical order.